microsetta_private_api/model/consent.py

```diff
@@ -0,0 +1,86 @@
+from datetime import datetime
+
+
+class ConsentDocument:
+    @staticmethod
+    def from_dict(input_dict, account_id, consent_id):
+        consent_type = input_dict["consent_type"]
+        locale = input_dict["locale"]
+        date_time = datetime.now()
+        consent = input_dict["consent"]
+        reconsent = input_dict["reconsent"]
+        return ConsentDocument(
+            consent_id, consent_type, locale,
+            date_time, consent, account_id,
+            reconsent
+        )
+
+    def __init__(self, consent_id, consent_type, locale,
+                 date_time, consent_content,
+                 account_id, reconsent):
+        self.consent_id = consent_id
+        self.consent_type = consent_type
+        self.locale = locale
+        self.date_time = date_time
+        self.consent_content = consent_content
+        self.account_id = account_id
+        self.reconsent = reconsent
+
+    def to_api(self):
+        result = {
+                    "consent_id": self.consent_id,
+                    "consent_type": self.consent_type,
+                    "locale": self.locale,
+                    "document": self.consent_content,
+                    "reconsent_required": self.reconsent
+                }
+
+        return result
+
+
+class ConsentSignature:
+    @staticmethod
+    def from_dict(input_dict, source_id, signature_id):
+        date_time = datetime.now()
+
+        parent_name_1 = input_dict.get("parent_1_name", None)
+
+        parent_name_2 = input_dict.get("parent_2_name", None)
+
+        deceased_parent = input_dict.get("deceased_parent", None)
+
+        assent_obtainer = input_dict.get("assent_obtainer", None)
```

Comment:
```suggestion
        parent_name_1 = input_dict.get("parent_1_name")
        parent_name_2 = input_dict.get("parent_2_name")
        deceased_parent = input_dict.get("deceased_parent")
        assent_obtainer = input_dict.get("assent_obtainer")
```
get() will return None by default if the key doesn't exist, no need to declare it.

Reply:
Corrected!